Tools/cases_generator/tier2_generator.py

```diff
@@ -64,6 +64,7 @@ def declare_variables(uop: Uop, out: CWriter) -> None:
 
 
 class Tier2Emitter(Emitter):
+
```

Comment:
Looks unrelated :)
```suggestion
```

Reply:
It is unrelated, but no harm in a little bit of tidying up.